src/encodings.h

```diff
@@ -139,23 +123,8 @@ typedef enum
 	GEANY_ENCODING_CP_932,
 
 	GEANY_ENCODINGS_MAX
-} GeanyEncodingIndex;
-
-/** Structure to represent an encoding to be used in Geany. */
-typedef struct
-{
-	/** The index of the encoding, must be one of GeanyEncodingIndex. */
-	gint   				 idx;
-	/** Internally used member for grouping */
-	gint   				 order;
-	/** Internally used member for grouping */
-	GeanyEncodingGroup   group;
-	/** String representation of the encoding, e.g. "ISO-8859-3" */
-	const gchar			*charset;
-	/** Translatable and descriptive name of the encoding, e.g. "South European" */
-	const gchar			*name;
-} GeanyEncoding;
```

Comment:
@codebrainz this breaks GeanyPy too.

Both are trivial to fix tho (just remove the wrappers).  Technically this struct *was* part of the API, but as there was nothing to get any instance of it removing the struct itself makes sense.